sw/device/tests/aon_timer_sleep_wdog_sleep_pause_test.c

```diff
@@ -45,8 +46,7 @@ bool test_main(void) {
                                               kDifToggleEnabled));
 
   // Check if there was a HW reset caused by the wdog bite.
-  dif_rstmgr_reset_info_bitfield_t rst_info;
-  CHECK_DIF_OK(dif_rstmgr_reset_info_get(&rstmgr, &rst_info));
+  dif_rstmgr_reset_info_bitfield_t rst_info = rstmgr_testutils_reason_get();
   CHECK_DIF_OK(dif_rstmgr_reset_info_clear(&rstmgr));
```

Comment:
This is already cleared by the ROM. You might want to use the testutil to clear the reset reason in retention SRAM

Reply:
Done.